src/main/java/cs2103/v15_1j/jimjim/ui/DayPickerPaneController.java

```diff
@@ -53,8 +55,9 @@ public BorderPane getDayPickerPane(){
 		return dayPickerPane;
 	}
 
-	public void refreshData(DataLists lists){
-		this.lists = lists;
+	public void refreshData(DataLists lists, DataLists displayLists){
+		this.masterList = lists;
+		this.displayList = displayLists;
```

Comment:
I think those 2 lines are not necessary. the list object will never change. Just the content will change so you don't need to update `masterList` and `displayList` to refer to itself.